### PR TITLE
Added use site variance annotation

### PR DESCRIPTION
**Pull request by gakuzzzz**
We would like to add `Map<String, String>` into `Request#fromParam`...

### CODE REVIEW

File: src/main/java/com/m3/curly/Request.java [hunk 1]
```diff
@@ -224,20 +224,22 @@ public String getContentType() {
         return getRequestBody().getContentType();
     }
 
+    @SuppressWarnings("unchecked")
     public Map<String, Object> getFormParams() {
-        return formParams;
+        return (Map<String, Object>) formParams;
     }
```
Comment (gakuzzzz) [inline src/main/java/com/m3/curly/Request.java]: By （good） rights, We should change this return type to `Map<String, ?>`

However I kept the compatibility.

**Comment by seratch** [comment]
LGTM. Could anyone merge this and release new version? if there is nobody can work on it, I'm willing to do it instead.

**Comment by gakuzzzz** [comment]
@seratch 
thanks! Currently, there is nobody who can release it.
Would you to do instead?

...and would you teach the way to m3dev members? cc/ @namutaka @reki2000

**Comment by seratch** [comment]
@gakuzzzz  How to add new developer to com.m3 publishers: https://issues.sonatype.org/browse/OSSRH-15265

**Comment by seratch** [comment]
>@gakuzzzz How to add new developer to com.m3 publishers: https://issues.sonatype.org/browse/OSSRH-15265

@reki2000 @namutaka @gakuzzzz (and others if needed) Sorry for bothering you. Please let me know your Sonatype JIRA account to publish libs. If you don't have Sonatype JIRA account, please create on this occasion.

**Comment by reki2000** [comment]
@seratch Thanks a lot. I created my Sonatype JIRA account as reki2000

**Comment by seratch** [comment]
I just released version 0.5.6 and wrote documentation for maintainers. https://github.com/m3dev/curly/blob/develop/maintainers-guide.md

Please ping me when all Sonatype JIRA accounts to grant are ready.

**Comment by gakuzzzz** [comment]
Thanks a lot!

I'll ask JIRA accounts to other members

**Pull request merged**